src/formik/ActionBlock.jsx

```diff
@@ -13,6 +13,7 @@ const ActionBlock = ({
   submitButtonProps,
   cancelButtonProps,
   isSubmitting: isFormSubmitting,
+  isOverlayComponent = false,
```

Comment:
@thomas-mathai  As we discussed in the call, right now we are making footer UI change only in the `Modal` component. So, this prop name is not correct because `Pane` is also an overlay and does not need the change introduced by this prop. Additionally, in issue https://github.com/bigbinary/neeto-ui/issues/2195 @abilash-sajeev  has requested adding a prop to reverse the button order in `ActionBlock`. Considering all this, I think instead of adding the prop `isOverlayComponent`, we can add a prop called `position` which will take two values, either `left` or `right`. By default, it will be `left`. For the `tertiary` style, we can use `cancelButtonProps` which is already there and by default the cancel button style will be `text`. 

So, when we use this `ActionBlock` component in `Modal`, we need to set `position` to `right` and `cancelButtonProps` style to `tertiary`.

cc: @praveen-murali-ind